readme.md

```diff
@@ -0,0 +1,15 @@
+# RangerQuiz
+
+RangerQuiz is an fun web app, through which you can enjoy quizzes of your favorite shows. In the end you will get to know the scores also. It consist of features as mentioned below.
+
+This web app is fully responsive.
+
+# Features
+
+1. [Home page](https://ranger-quiz.netlify.app/)
+2. [Quiz category page](https://ranger-quiz.netlify.app/quiz-category-page.html)
+3. [Rules page](https://ranger-quiz.netlify.app/rules-page.html)
+4. [Question page](https://ranger-quiz.netlify.app/question-page.html)
+5. [Result page](https://ranger-quiz.netlify.app/answer-page.html)
+
+[Animation](/gif/quiz-web-app.gif)
```

Comment:
Add the link as `![Animation](/gif/quiz-web-app.gif)` so that it comes as a gif in Readme not as a link.

Tip: Directly edit it on github so that the gif you upload from here will be stored on github servers, not inside your codebase.

Reply:
okay

Reply:
Good!